### PR TITLE
[16.0][UPD] sale_order_import_edifact: Update EDIFACT mapping

### PR DESCRIPTION
**Pull request by thienvh332**

### CODE REVIEW

File: sale_order_import_edifact/tests/common.py [hunk 1]
```diff
@@ -52,6 +52,7 @@ def get_test_data_no_ean_in_lin(env):
                 ),
                 "client_order_ref": "467819",
                 "date_order": "2023-03-20",
+                "comitment_date": "2023-03-22 00:00:00",
```
Comment (simahawk) [inline sale_order_import_edifact/tests/common.py]:
```suggestion
                "commitment_date": "2023-03-22 00:00:00",
```

File: sale_order_import_edifact/tests/test_edifact_order_import.py [hunk 2]
```diff
@@ -61,6 +61,9 @@ def test_edifact_order_import_no_ean_in_lin(self):
             so = self.env["sale.order"].browse(action["res_id"])
             self.assertEqual(so.partner_id, expected["partner"])
 
+            if expected.get("commitment_date"):
+                self.assertEqual(so.commitment_date, expected["commitment_date"])
```
Comment (simahawk) [inline sale_order_import_edifact/tests/test_edifact_order_import.py]: not tested because of the typo

Comment (thienvh332) [reply sale_order_import_edifact/tests/test_edifact_order_import.py]: Thanks @simahawk , I updated it.

**Review by simahawk — APPROVED** [review]

**Pull request merged**